src/client/v2/algod/algod.ts

```diff
@@ -462,4 +464,36 @@ export default class AlgodClient extends ServiceClient {
   getProof(round: number, txID: string) {
```

Comment:
This endpoint now has one more optional parameter, I think we should reflect that as well.
In addition, even though the rest endpoint url itself (`/v2/blocks/${this.round}/transactions/${this.txID}/proof`) stays the same  I think it is better to rename it to `getTransactionProof` or similar since it the name doesn't suggest what proof you get.